packages/sdk/src/swap/types.ts

```diff
@@ -209,6 +209,12 @@ type SwapState =
       broadcastSucceededBlockIndex: string;
       broadcastTransactionRef: string;
     } & BroadcastRequested)
+  | ({
+      state: 'REFUNDED';
+      broadcastSucceededAt: number;
+      broadcastSucceededBlockIndex: string;
+      broadcastTransactionRef: string;
+    } & BroadcastRequested)
```

Comment:
i dont think you can use `BroadcastRequested` because it has `swapExecutedAt`

Reply:
yeah, that's true
updated